credsweeper/filters/value_jfrog_token_check.py

```diff
@@ -28,10 +31,19 @@ def run(self, line_data: LineData, target: AnalysisTarget) -> bool:
             True, if need to filter candidate and False if left
 
         """
-        if not line_data.value:
+        value = line_data.value
+        if not value:
             return True
         with contextlib.suppress(Exception):
-            decoded = Util.decode_base64(line_data.value, padding_safe=True, urlsafe_detect=True)
-            if self._pattern.match(decoded.decode(ASCII)):
-                return False
+            if value.startswith("cmVmdGtuO"):
+                decoded = Util.decode_base64(value, padding_safe=True, urlsafe_detect=True)
+                if self._pattern.match(decoded.decode(ASCII)):
+                    # identity token
+                    return False
+            if value.startswith("AKCp"):
```

Comment:
As I checked the link the prefix from JFrog was `AKCp8`..
Does it wrong? Actually I already saw other token which you sent by email which starts with `AKCpB`.
If you have other reason to use `AKCp` as prefix, please let me know.

Reply:
Yes, i found 3 variants. 2,8,B. Anyway, the "if" is for detecting type of token only. It distinguishes "cmVmdGtuO" prefix and "AKCp".